src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const result = {};
+
+  for (const key in robot) {
+    const Key = robot[key];
```

Comment:
why `Key`?
I see you have `key` in the loop so create another name for this variable without uppercase